flink-state-backends/flink-statebackend-forst/src/main/java/org/apache/flink/state/forst/fs/ForStFlinkFileSystem.java

```diff
@@ -42,10 +44,16 @@ public class ForStFlinkFileSystem extends FileSystem {
     // TODO: make it configurable
     private static final int DEFAULT_INPUT_STREAM_CAPACITY = 32;
 
+    private static final String SST_SUFFIX = ".sst";
+
     private final FileSystem delegateFS;
+    private final FileSystem localFS;
```

Comment:
I'd suggest a class member of local base path, to prevent frequent map accessing.